### DISCUSSION TITLE
Been having issues with entering the cookie

### DISCUSSION THREAD

**Discussion by LitoStarr** [UltimaHoarder/UltimaScraper#1396] [Q&A]
It seem OnlyFans.com has updated their cookies and when I copy them into the .profile file, it's still not being authenticated.

I'm using MacOS. Is anyone else having this problem?

**Comment by fightcancer2** [comment]
Yes.

**Reply by LitoStarr** [reply]
Were you ever able to solve it? I went back to v7.2

**Reply by fightcancer2** [reply]
No.  I can't get 7.4 to work.
